scania-digital-design-system/tegel
# test(accordion): update unit tests

**Describe pull-request**  
Update unit tests for the Accordion component in accordance with unit test scenarios.

**Solving issue**  
Fixes: [CDEP-3025](https://tegel.atlassian.net/browse/CDEP-3025)

**How to test**  
**How to test**  
1. Go to accordion [unit test spreadsheet](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/_layouts/15/Doc.aspx?sourcedoc=%7BCA48BF6D-92CF-4E8C-8F45-6337E2002EC4%7D&file=Unit%20test%20scenarios.xlsx&nav=MTVfezAwMDAwMDAwLTAwMDEtMDAwMC0wMDAwLTAwMDAwMDAwMDAwMH0&action=default&mobileredirect=true&wdOrigin=TEAMS-ELECTRON.p2p_ns.bim&wdExp=TEAMS-CONTROL&wdhostclicktime=1704804632230&web=1).
2. Check that code corresponds to requirements from spreadsheet
3. Run test locally to see if tests are passing, try to change something in index.html that will break test

[CDEP-3025]: https://tegel.atlassian.net/browse/CDEP-3025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
